tools/manifest-templator/manifest-templator.go

```diff
@@ -240,6 +240,7 @@ func main() {
 	imageName := flag.String("image-name", components.Name, "The operator image's name")
 	containerTag := flag.String("container-tag", "latest", "The operator image's container tag")
 	imagePullPolicy := flag.String("image-pull-policy", "Always", "The pull policy to use on the operator image")
+	allowMultus := flag.Bool("allow-multus", true, "Install Multus RBAC, making it possible to deploy Multus through the operator (should be disabled on OpenShift to limit CNAO's RBAC to the necessary minimum)")
```

Comment:
PR desc does says what happen with and without the flag but
worth to mention please on PR desc that Openshift must disable it
as this comment says